nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/PostgreSQLBackend.java

```diff
@@ -61,7 +64,19 @@ public Map<String, POSTGRESQL_COLUMN_TYPES> listOfFields(String attrPersistence,
                 String attrName = encodeAttributeToColumnName(attribute.getAttrName(), attribute.getDatasetId(), datasetIdPrefixToTruncate);
                 if (attribute.getAttrValue() instanceof Number) {
                     if (aggregation.replace(attrName, POSTGRESQL_COLUMN_TYPES.NUMERIC) == null)
-                        aggregation.putIfAbsent(attrName, POSTGRESQL_COLUMN_TYPES.NUMERIC);
+                    aggregation.putIfAbsent(attrName, POSTGRESQL_COLUMN_TYPES.NUMERIC);
+                } else if ("GeoProperty".equals(attribute.getAttrType())) {
```

Comment:
aren't you missing the brute value of the `coordinates` attribute?

Reply:
That's to say ? I did not understand very well ?

Reply:
I don't see where you store the brute value of the `coordinates` attribute (e.g., `[3.21,47.32]`)

Reply:
I get the coordinates here (when it's Point) https://github.com/easy-global-market/fiware-draco/blob/ff49fb90003cdfc5354b6259efb5707f1a86022a/nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/PostgreSQLBackend.java#L253

Reply:
The value of the `location` attribute remains a geojson which allows me to pass the geojson directly when I want and to retrieve only the coordinates part